modules/shared/system_module_version.mk

```diff
@@ -36,6 +41,21 @@ ifeq ($(PLATFORM_ID),32)
 PREBOOTLOADER_PART1_DEPENDENCY = 3
 endif
 
+# The version of the bootloader that the system firmware requires for 28
+# NOTE: this will force the device into safe mode until this dependency is met, which is why
+# this version usually lags behind the current bootloader version, to avoid non-mandatory updates.
+ifeq ($(PLATFORM_ID),28)
```

Comment:
Same, let's not split versions/dependencies between p2/trackerm for now unless required. We can always add a separate depenendency later when needed.